packages/katello/katello/katello.spec

```diff
@@ -194,6 +194,9 @@ Useful utilities for managing Katello services
 %{_sysconfdir}/bash_completion.d/katello-service
 
 %changelog
+* Tue Aug 13 2019 Lukas Zapletal <lzap+rpm@redhat.com> - 3.14.0-1.1.master
```

Comment:
It seems the release went out of sync with the version indicated in the changelog, possibly when 3.14 was bumped. @evgeni should this be 3.14.0-2.1.master now?

Reply:
-0.2.master

Reply:
Updated the changelog